src/migrations/20241125115848-add-enabled-in-oss-to-project-and-environment.js

```diff
@@ -0,0 +1,16 @@
+exports.up = function(db, cb) {
+    db.runSql(`
+        ALTER TABLE environments ADD COLUMN enabled_in_oss BOOLEAN NOT NULL DEFAULT false;
+        ALTER TABLE projects ADD COLUMN enabled_in_oss BOOLEAN NOT NULL DEFAULT false;
```

Comment:
Hacking this would not be difficult... just run a query against the db changing all enabled_in_oss to true... it adds some complexity though

Reply:
And we hide default by default these days, so it's really only development and production.
Yeah ok, let's be explicit then. Updating